editor/src/components/canvas/canvas-utils.ts

```diff
@@ -2967,13 +2967,14 @@ export function getValidElementPathsFromElement(
   projectContents: ProjectContentTreeRoot,
   filePath: string,
   uiFilePath: string,
-  parentIsScene: boolean,
+  isOnlyChildOfScene: boolean,
   parentIsInstance: boolean,
   transientFilesState: TransientFilesState | null,
   resolve: (importOrigin: string, toImport: string) => Either<string, string>,
 ): Array<ElementPath> {
   if (isJSXElementLike(element)) {
     const isScene = isSceneElement(element, filePath, projectContents)
+    const isSceneWithOneChild = isScene && element.children.length === 1
```

Comment:
This just means that the element has a single static child - it can still have multiple dynamic children if e.g. that child is a map expression returning elements.
And I think regarding selection the number of rendered children what matters.